java/.gitignore

```diff
@@ -14,3 +14,5 @@ target
 
 # don't catch config.yaml, may contain secret salt
 impl/cmd-line/config.yaml
+
+.idea/
```

Comment:
I would remove this from .gitignore, because this folder was already committed before. But for main project we've excluded `.idea` folder: https://github.com/Worklytics/evalengin/blob/working/.gitignore